src/components/calltree/CallTree.js

```diff
@@ -272,6 +272,5 @@ export default explicitConnect<{||}, StateProps, DispatchProps>({
     changeExpandedCallNodes,
     addTransformToStack,
   },
-  options: { withRef: true },
```

Comment:
Seemed to be unused so I simply removed it. (Any code using this should've called `getWrappedInstance()`)